cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/ConsentStatusApiModel.kt

```diff
@@ -113,31 +150,63 @@ data class USNatConsentData(
 
 @Serializable
 data class GdprCS(
-    val applies: Boolean?,
-    @SerialName("categories") val categories: List<String>?,
-    @SerialName("consentAllRef") val consentAllRef: String?,
-    @SerialName("consentedToAll") val consentedToAll: Boolean?,
-    @SerialName("legIntCategories") val legIntCategories: List<String>?,
-    @SerialName("legIntVendors") val legIntVendors: List<String>?,
-    @SerialName("postPayload") val postPayload: PostPayload?,
-    @SerialName("rejectedAny") val rejectedAny: Boolean?,
-    @SerialName("specialFeatures") val specialFeatures: List<String>?,
-    @SerialName("vendors") val vendors: List<String>?,
-    @SerialName("addtlConsent") val addtlConsent: String?,
-    @SerialName("consentStatus") val consentStatus: ConsentStatus?,
-    @SerialName("cookieExpirationDays") val cookieExpirationDays: Int?,
-    @SerialName("customVendorsResponse") val customVendorsResponse: CustomVendorsResponse?,
-    @SerialName("dateCreated") val dateCreated: String?,
-    @SerialName("euconsent") val euconsent: String?,
-    @Serializable(with = GrantsSerializer::class) val grants: Map<String, GDPRPurposeGrants>?,
-    @Serializable(with = JsonMapSerializer::class) val TCData: Map<String, JsonElement>?,
-    @SerialName("localDataCurrent") val localDataCurrent: Boolean?,
-    @SerialName("uuid") var uuid: String?,
-    @SerialName("vendorListId") val vendorListId: String?,
+    val applies: Boolean? = null,
+    @SerialName("categories") val categories: List<String>? = emptyList(),
+    @SerialName("consentAllRef") val consentAllRef: String? = null,
+    @SerialName("consentedToAll") val consentedToAll: Boolean? = null,
+    @SerialName("legIntCategories") val legIntCategories: List<String>? = emptyList(),
+    @SerialName("legIntVendors") val legIntVendors: List<String>? = emptyList(),
+    @SerialName("postPayload") val postPayload: PostPayload? = null,
+    @SerialName("rejectedAny") val rejectedAny: Boolean? = null,
+    @SerialName("specialFeatures") val specialFeatures: List<String>? = emptyList(),
+    @SerialName("vendors") val vendors: List<String>? = emptyList(),
+    @SerialName("addtlConsent") val addtlConsent: String? = null,
+    @SerialName("consentStatus") val consentStatus: ConsentStatus? = null,
+    @SerialName("cookieExpirationDays") val cookieExpirationDays: Int? = null,
+    @SerialName("customVendorsResponse") val customVendorsResponse: CustomVendorsResponse? = null,
+    @SerialName("dateCreated") val dateCreated: String? = null,
+    @SerialName("euconsent") val euconsent: String? = null,
+    @Serializable(with = GrantsSerializer::class) val grants: Map<String, GDPRPurposeGrants>? = emptyMap(),
+    @Serializable(with = JsonMapSerializer::class) val TCData: Map<String, JsonElement>? = emptyMap(),
```

Comment:
The same concern as above. In this class it's regarding `categories`, `legIntCategories`, `legIntVendors` , `specialFeatures` , `vendors` , `grants` , `TCData`